### ISSUE TITLE
A11y_.NETCore_RecipesAppRecipesSearch_Searchbymealtypesection_Resize: After resizing the font size to largest name of the "Meal types" present under 'Search by meal type' are not visible.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15228]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_Android_May23; #.NET Core; #WCAG1.4.4; #Android; #AndroidPhone;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: Android 13
Screen Reader: Talkback

### Pre- requisite:
Settings-> Accessibility-> Text and Display -> Font Size-> Largest

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "Search by meal type section".
3. Observe the issue

### Actual Results:
After resizing the font size to largest name of the "Meal types" present under 'Search by meal type' are not visible.

### Expected Results:
After resizing the font size to largest name of the "Meal types" present under 'Search by meal type' should be clearly visible.

### User Impact:
Low vision users will not be able to see the name of the "Meal types" present under 'Search by meal type' after resizing the font size to largest.

### Attachment:
**Issue:**
![Issue 2](https://github.com/dotnet/maui/assets/90901519/5172e096-1ba7-491b-a859-9522a688a746)

**Actual Image:**
![issue 3 orogin](https://github.com/dotnet/maui/assets/90901519/0a79ff86-5aff-4f90-a08c-02f7718209b5)

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 13

**Comment by rachelkang** [comment]
Hi, @PoojaNamde - thanks for reporting this issue. I tried to repro what you described but was unable to. Could you let me know if this video looks right to you? If it does, could you share what Android device model you tested on? If it doesn't, could you let me know what I might be doing wrong? Thanks!

https://github.com/dotnet/maui/assets/21988533/9bc38a73-cc78-4c8f-bbf8-d7277d72af15

**Comment by PoojaNamde** [comment]
@rachelkang 

The video is correct. We are using Google pixel 5 (Version Android 13).

**Comment by rachelkang** [comment]
Thanks. I tested on Android 12, so will update to Android 13 to see if that changes anything.

**Comment by rachelkang** [comment]
Hi, @PoojaNamde - we were still unable to reproduce this issue. Would you be able to provide more info? Perhaps a video?

**Comment by PoojaNamde** [comment]
@rachelkang 

To reproduce this issue we only changed the font size to largest. We will reverify this issue at our end to check whether if it is still reproducing or not.

**Comment by PoojaNamde** [comment]
#Regressed:07-03-23; #Closed

Reverified this issue in the application, issue is not reproducing, hence closing the bug. Please find below screenshot for reference.

![15228_Closed](https://github.com/dotnet/maui/assets/90901519/c2ebce89-e1e2-49f5-ac05-f23ddbee0450)
